message ClientRobot {
  required fixed32 id = 1;
  //required float velocityx = 2;
  //required float velocityy = 3;
  //optional float angle = 4;     // in radians
  required bytes vel = 2;
  optional bool puckPickup = 3; // true - try to get a puck,
                                // false - try to drop a puck
}

message BouncedRobot {
  required ClientRobot clientrobot = 1;
  required uint32 bounces = 2;
}
